cluster-install/install.sh

```diff
@@ -2617,16 +2713,26 @@ if trident_will_be_installed_or_modified; then
             if trident_image_needs_upgraded; then
                 if config_trident_image_is_custom || prompt_user_yes_no "Would you like to upgrade Trident?"; then
                     step_generate_torc_patch "$_EXISTING_TORC_NAME" "$(get_config_trident_image)" "" "" "$(get_config_trident_autosupport_image)"
-                    trident_operator_image_needs_upgraded && step_generate_trident_operator_patch
+                    if trident_operator_image_needs_upgraded; then
+                        step_generate_trident_operator_patch
+                        _USER_ACCEPTED_TRIDENT_OPERATOR_UPGRADE="true"
+                    fi
                 else
-                    loginfo "Trident will not be upgraded."
+                    _msg="You have chosen to use a version of Trident that is not supported with the current version"
+                    _msg+=" of Astra Control. This may result in some App Data Management operations not functioning"
+                    _msg+=" correctly or being blocked within Astra Control. It is highly recommended to upgrade"
+                    _msg+=" Trident to ensure compatibility and proper functionality."
+                    logwarn "$_msg"
```

Comment:
This is a new warning message for users declining Trident upgrades, as requested by Vijitha.